### DISCUSSION TITLE
Should Quantumix to be full duplex?

### DISCUSSION THREAD

**Discussion by fu050409** [noctisynth/Quantumix#27] [Polls]
There will be less overhead if we use full duplex protocol [Oblivion](https://github.com/noctisynth/oblivion-rust), also. the service of Quantumix provided will be private and safe. But this may cause more barriers and more time to develop.

**Comment by fb0sh** [comment]
> There will be less overhead if we use full duplex protocol [Oblivion](https://github.com/noctisynth/oblivion-rust), also. the service of Quantumix provided will be private and safe. But this may cause more barriers and more time to develop.


I approve it.

**Comment by HsiangNianian** [comment]
> There will be less overhead if we use full duplex protocol [Oblivion](https://github.com/noctisynth/oblivion-rust), also. the service of Quantumix provided will be private and safe. But this may cause more barriers and more time to develop.

no matter

**Comment by fu050409** [comment]
OK, we'll use full duplex Oblivion instead.
